client-participation-alpha/src/pages/[conversation_id].astro

```diff
@@ -23,56 +24,64 @@ if (!conversation_id ||
   return Astro.redirect('/404');
 }
 
+// Load translations first so we can use them in error messages
+const s = await getTranslations();
+
 let initialData;
 let fetchError = null;
 
 try {
-  const apiParams = new URLSearchParams();
-  if (conversation_id) {
-    apiParams.append('conversation_id', conversation_id);
-  }
-
   const xid = Astro.url.searchParams.get('xid');
   const x_name = Astro.url.searchParams.get('x_name');
   const x_profile_image_url = Astro.url.searchParams.get('x_profile_image_url');
 
+  const apiParams: Record<string, any> = {
+    conversation_id,
+    includePCA: false
+  };
 
   if (xid) {
-    apiParams.append('xid', xid);
+    apiParams.xid = xid;
   }
   if (x_name) {
-    apiParams.append('x_name', x_name);
+    apiParams.x_name = x_name;
   }
   if (x_profile_image_url) {
-    apiParams.append('x_profile_image_url', x_profile_image_url);
+    apiParams.x_profile_image_url = x_profile_image_url;
   }
 
-
-  initialData = await PolisNet.polisGet('/participationInit', { conversation_id, includePCA: false });
+  initialData = await PolisNet.polisGet('/participationInit', apiParams);
```

Comment:
PCA cannot be in here, perf is too bad

Reply:
it's set up that way, with `includePCA: false`
https://github.com/compdemocracy/polis/pull/2279/files/2c64d2b5a16652feb84f3bffac9dc3df264e4bb9#diff-929636c83f42520fd7300a5c314cdf61cd33e35c46f6c9e08a181a5746dd4201R38-R41